.github/workflows/playwright.yml

```diff
@@ -0,0 +1,68 @@
+name: Playwright Tests
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    strategy: 
+      matrix:
+        node-version: 
+          # - '12'
```

Comment:
The commented out versions had been failing while I was working on this PR. Since the PR has been stable I haven't retried them in CI yet. I'll push up a quick commit to see if they're still failing before signing off for now

Reply:
Confirmed: `v12.x` and `v14.x` are failing in CI. I'm getting an error related to the `ejs` package dependency: https://github.com/uchilaka/audit-export/actions/runs/8521632747/job/23340185623?pr=2